week-4/library_management_system/src/main/java/job_hacker/tasks/dtos/BookDto.java

```diff
@@ -0,0 +1,50 @@
+package job_hacker.tasks.dtos;
+
+import job_hacker.tasks.models.enums.AvailabilityStatus;
+
+public class BookDto {
+    private int ID;
```

Comment:
we always recommend to use String types to represent the id to avoid overflow problems